Carbon.Registry/Functions/Get-CRegistryKeyValue.ps1

```diff
@@ -0,0 +1,42 @@
+
+function Get-CRegistryKeyValue
+{
+    <#
+    .SYNOPSIS
+    Gets the value from a registry key.
+
+    .DESCRIPTION
+    PowerShell's `Get-ItemProperty` cmdlet is a pain to use.  It doesn't actually return an object representing a

```

Comment:
Focus the description more on what the function does for the user, not why it does it? The "why" should belong in the README?